frontend/packages/operator-lifecycle-manager/src/components/catalog-source.tsx

```diff
@@ -420,6 +448,12 @@ const DisabledPopover: React.FC<DisabledPopoverProps> = ({ operatorHub, sourceNa
   );
 };
 
+const getRegistryPollInterval = (catalogSource: CatalogSourceKind): string => {
+  return catalogSource.spec?.updateStrategy
+    ? catalogSource.spec.updateStrategy.registryPoll?.interval
+    : undefined;
```

Comment:
Should have caught this on last review, but this is the same as

```suggestion
  return catalogSource.spec?.updateStrategy?.registryPoll?.interval;
```